convert_to_our_format.py

```diff
@@ -0,0 +1,50 @@
+import json
+
+aliasesFields = ['preferred synonym', 'synonyms', 'preferred fully specified name', 'fully specified names']
+f = open('snomed/concepts.json')
+concept_dict = json.load(f)
+f = open('snomed/is_a.json')
+is_a_dict = json.load(f)
+f = open('snomed/concept_without_father.json')
+concept_without_father = json.load(f)
+id = 1
+
+
+def create_hierarchy(conc_id):
+    global id
+    id = id + 1
+    if conc_id not in is_a_dict:
+        aliases = {}
+        for alias in aliasesFields:
+            if len(concept_dict[conc_id][alias]) > 0:
+                aliases[alias] = concept_dict[conc_id][alias]
+        concept_leaf = {"name": concept_dict[conc_id]["name"], "id": id, "state": {"aliases": aliases}, "children": []}
+        return concept_leaf
+
+    aliases = {}
+    for alias in aliasesFields:
+        if len(concept_dict[conc_id][alias]) > 0:
+            aliases[alias] = concept_dict[conc_id][alias]
+    concept = {"name": concept_dict[conc_id]["name"], "id": id, "state": {"aliases": aliases}, "children": []}
+
+    children = is_a_dict[conc_id]
+    for c in children:
+        concept["children"].append(create_hierarchy(c))
+
+    return concept
+
+
+def create_our_format():
```

Comment:
its a bit confusing that you call this file convert and the function create. which is it?
you can choose either, but (a) be consistent, (b) if you go with `convert` it means you need to return a value from this function (convert implies you get x and return a converted y) and not create here anything (leave that to the main file/function) but if you choose to go with `create` then its ok as you did.